app/__init__.py

```diff
@@ -22,23 +13,17 @@ def create_app():
     # initialize configuration values
     config_name = os.getenv("FLASK_ENV", 'default')
     app.logger.debug("Reading " + config_name + " configuration...")
+    app.logger.debug("App config:")
 
-    config_obj = CONFIG[config_name]
-    app.logger.debug("config obj: " + config_obj)
-    app.config.from_object(config_obj)
-
-    if app.config.from_pyfile('flask_config.py'):
-        app.logger.info("Successfully read instance configuration file.")
-    app.logger.info("Flask config:")
-    app.logger.info(app.config)
+    # hard-code some configuration
+    app.config["BASE_CLASSIFIER_DIR"] = "./classifier_models"
```

Comment:
will local.py have access to this? maybe make actual global variable?

Reply:
nah, but it has a default flag value of the same. if we find that they need to match for some reason later (for now, it shouldn't matter) we can change to import from one global config.